.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: ci
+on:
+  push:
+    branches:
+      - "**"
+  workflow_dispatch:
+    inputs:
+      deploy-to-dev:
+        description: "Whether to deploy to dev"
+        type: boolean
+        required: true
+        default: false
+
+defaults:
+  run:
+    # NOTE: A bit stricter than the default bash options used by GitHub Actions
+    # (bash --noprofile --norc -e -o pipefail {0})
+    shell: bash --noprofile --norc -euo pipefail {0}
+
+# NOTE: Set concurrency for the current workflow to 1
+concurrency: ci-${{ github.ref }}-${{ github.workflow }}
+
+jobs:
+  build-and-deploy:
+    timeout-minutes: 60
+    runs-on: ubuntu-22.04
+    permissions:
+      actions: read
+      contents: read
+      id-token: write
+    steps:
+
+      # - name: disable job
```

Comment:
Should this instead be a conditional step that only runs if the name of the repository is something else? `if: ${{ github.event.repository.name != 'liflig-cdk-app-reference' }}`

Reply:
Yeah probably, makes it less annoying to test in this repo. Fixing

Reply:
https://github.com/capralifecycle/liflig-cdk-app-reference/pull/64/commits/b195e6ab491ad850537e41b3aa820001d00ae791